src/app/containers/Image/helpers/srcSetHelper.js

```diff
@@ -0,0 +1,14 @@
+const createSrcset = (originCode, locator) => {
+  const resolutions = [240, 320, 480, 624, 800];
+  if (originCode === 'pips') {
+    return null;
+  }
+  const urls = resolutions.map(
```

Comment:
If the original image width is smaller than some of these resolutions, we shouldn't render the value in the srcset, otherwise it'll mean an iChef recipe will be applied to make an image file that is larger than the original. 

So if the original image width is 400px, the srcset should only include urls for `240` &  `320` widths.

Reply:
@sareh How can I request the original width?

Reply:
The original width is passed into the Image Container in the data - it's the `width` property defined here: https://github.com/bbc/simorgh/pull/1683/files#diff-445cfd09f5ef4bfbe3544be7b7f2da00R52 you can pass it in as `originalImageWidth` to this function and use it here.